packages/build/src/adapter/deno/index.ts

```diff
@@ -0,0 +1,37 @@
+import type { Plugin } from 'vite'
+import type { BuildOptions } from '../../base.js'
+import buildPlugin from '../../base.js'
+import { serveStaticHook } from '../../entry/serve-static.js'
+
+export type DenoBuildOptions = {
+  staticRoot?: string | undefined
+} & BuildOptions
+
+const denoBuildPlugin = (pluginOptions?: DenoBuildOptions): Plugin => {
+  return {
+    ...buildPlugin({
+      ...{
+        entryContentBeforeHooks: [
+          async (appName, options) => {
+            // eslint-disable-next-line quotes
+            let code = "import { serveStatic } from 'hono/deno'\n"
+            code += serveStaticHook(appName, {
+              filePaths: options?.staticPaths,
+              root: pluginOptions?.staticRoot,
+            })
+            return code
+          },
+        ],
+        entryContentAfterHooks: [
+          async (appName) => {
+            return `Deno.serve(${appName}.fetch)`
```

Comment:
Using `entryContentAfterHooks` is a good idea, but I think we don't have to use `Deno.serve`; the user can run it with the `deno serve` command without it. If we use `Deno.serve`, we can't specify the port number and other options with this place. You can make them options, but they will be verbose.

Reply:
I agree. I changed.